tool-plugins/vscode/package-lock.json

```diff
@@ -257,7 +257,7 @@
     },
     "async": {
       "version": "1.2.1",
-      "resolved": "https://registry.npmjs.org/async/-/async-1.2.1.tgz",
+      "resolved": "http://registry.npmjs.org/async/-/async-1.2.1.tgz",
```

Comment:
Mistakenly committed.

Reply:
I think these files are committed with merge. Will re-merge and push